### ISSUE TITLE
a11y issues and fixes

### ISSUE THREAD

**Issue by InnaAtanasova** [SAP/fundamental-ngx#6497]
When an a11y issue is created and it states that some changes/fixes are required, it should also contain explanation why it should be done AND/OR a link to some official a11y documentation where this is explained in details. 
@droshev to meet with Suren and Manav to discuss this in more details

@mikerodonnell89 @InnaAtanasova
